locales/it/index.yml

```
@@ -119,7 +119,7 @@ global:
     dayMonth: "DD/MM"
     dayMonthYear: "D MMMM YYYY"
     weekdayDayMonthYear: "dddd D MMMM YYYY"
-  genericError: Ooops, qualcosa è andato storto, riprova per piacere!
+  genericError: There is a temporary problem, please try again.
```

It should be italian 🇮🇹

---

```suggestion
  genericError: C'è un problema temporaneo, riprova.
```

Copy and pasted the translation on both files: Translation is now coming home! 😅